Octokit/Clients/RepositoriesClient.cs

```diff
@@ -50,6 +50,7 @@ public RepositoriesClient(IApiConnection apiConnection) : base(apiConnection)
         /// <param name="newRepository">A <see cref="NewRepository"/> instance describing the new repository to create</param>
         /// <exception cref="ApiException">Thrown when a general API error occurs.</exception>
         /// <returns>A <see cref="Repository"/> instance for the created repository.</returns>
+        [Preview("mercy")]
```

Comment:
⚠️ Realizing that I'm not sure I actually needed to add the mercy preview header given that it was in 2017 and it may very well be GA right now. 

Anyone have a definitive answer on this? I'm less familiar with how the GH API treats these things.

Reply:
It's still listed as preview https://docs.github.com/en/rest/overview/api-previews#repository-topics so this is fine